io.github.maurycyliebner.enve.yaml

```diff
@@ -0,0 +1,186 @@
+      commit: 4cd7fe1509f6e8b7a60430c6c19f2f03159d7889
+
+- name: skia
+  buildsystem: simple
+  build-commands:
+    - gn gen out/Release --args='is_official_build=true
+                                 is_debug=false
+                                 is_component_build=false
+                                 target_os="linux"
+                                 target_cpu="x64"
+                                 skia_use_dng_sdk=false
+                                 skia_use_system_libjpeg_turbo=false
+                                 skia_use_libwebp_decode=false
+                                 skia_use_libwebp_encode=false
+                                 skia_use_zlib=true'
+    - ninja -C out/Release skia
+    - mkdir /app/skia_root
+    - cp -r * /app/skia_root
+  cleanup: ["*"]
+  sources:
+    - type: git
+      dest: third_party/externals/sfntly
+      url: https://github.com/googlefonts/sfntly.git
+      commit: 1e7adf313bd9c488a70015f8df8782f7c65e9ce7
+    - type: archive
+      dest: third_party/externals/libjpeg-turbo
+      url: https://downloads.sourceforge.net/project/libjpeg-turbo/2.0.4/libjpeg-turbo-2.0.4.tar.gz
+      sha256: 33dd8547efd5543639e890efbf2ef52d5a21df81faf41bb940657af916a23406
+    - type: git
+      # url: https://github.com/google/skia.git
+      url: https://skia.googlesource.com/skia
+      commit: 76f4fff613b5ba4365d24dbd943a8ed47221f1e9
+
+- name: enve
+  buildsystem: simple
+  build-options:
+    env:
+      QMAKEPATH: /app/lib
+  build-commands:
+    - mkdir _build && cd _build &&
+      qmake -r .. CONFIG+=release
+                  CONFIG+=force_debug_info
+                  INCLUDEPATH+=/app/include/QtWebEngineWidgets
+                  INCLUDEPATH+=/app/skia_root
+                  INCLUDEPATH+=/app/gperftools_root/src
+                  LIBS+=-L/app/lib
+                  LIBS+=-L/app/skia_root/out/Release
+                  LIBS+=-L/app/gperftools_root &&
+      sed -e 's|/usr/lib/x86_64-linux-gnu/libQt5WebEngine|/app/lib/x86_64-linux-gnu/libQt5WebEngine|g'
+          -i src/app/Makefile &&
+      make -j $FLATPAK_BUILDER_N_JOBS &&
+      cp -v src/app/enve /app/bin &&
+      cp -v src/core/libenvecore.so.1 /app/lib
+    - install -Dm644 build/AppDir/usr/share/icons/hicolor/48x48/enve.png -t /app/share/icons/hicolor/48x48/apps
+    - install -Dm644 build/AppDir/usr/share/icons/hicolor/256x256/enve.png -t /app/share/icons/hicolor/256x256/apps
+    - install -Dm644 build/AppDir/usr/share/icons/hicolor/scalable/enve.svg -t /app/share/icons/hicolor/scalable/apps
+    - install -Dm644 build/AppDir/usr/share/applications/enve.desktop -t /app/share/applications
+    - install -Dm644 io.github.maurycyliebner.enve.metainfo.xml -t /app/share/metainfo
+  sources:
+    - type: file
+      path: io.github.maurycyliebner.enve.metainfo.xml
+    - type: git
+      # https://github.com/flatpak/flatpak-builder/issues/337
+      # disable-submodules: true
+      url: https://github.com/MaurycyLiebner/enve.git
```

Comment:
This has to point to specific commit, and have a way to be updated automatically, unless you plan to handle it yourself.

Reply:
fixes